src/dmd/globals.d

```diff
@@ -208,6 +208,11 @@ struct Param
     const(char)* moduleDepsFile;        // filename for deps output
     OutBuffer* moduleDeps;              // contents to be written to deps file
 
+    bool doMakeDeps;                    // generate make-style dependencies file
+    const(char)* makeDepsFile;          // filename for make-style dependencies output
+    const(char)* makeDepsDir;           // directory for make-style dependencies output
+    const(char)* makeDepsTarget;        // the make target name
```

Comment:
Can these `const(char)*` fields be made into D strings, i.e. `const(char)[]`? On the C++ side an ABI compatible struct can be used.

Reply:
As much as I'd like this to happen, I don't think it's fair to ask a contributor to break the current status quo.
Such a change should be done on its own.